recipes/font-ttf-dejavu-sans-mono-feedstock/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = '2.37' %}
+
+package:
+  name: font-ttf-dejavu-sans-mono
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/dejavu/files/dejavu/{{ version }}/dejavu-sans-ttf-{{ version }}.zip
+  sha256: 5c6e497a2f36552cb5ffb112c413a6af39c0f3c47653662b90b4fa6499822fd7
+
+build:
+  number: 0
+  noarch: generic
+
+# test:
+#   commands:
+#     - test -f ${PREFIX}/fonts/DejaVuSans.ttf  # [unix]
```

Comment:
I think you could write a noarch test with Python (or xonsh) that detected if the file exists.

Reply:
Why would you introduce such heavy dependencies to check if a file exists when the shell can do it just fine?

Reply:
Tests can have selectors in noarch recipes.
Btw, this is commented because of an issue in conda-build

From PR description,

> Anybody has ideas how to fix the tests? They fail with the same error as conda/conda-build#3553

Reply:
Yes, tests are deactivated because of the reverenced bug. Afaik that bug happens, when there are no run requirements. The selector in the test case seems not to be the issue.

Reply:
Ok, found a workaround by adding a dummy requirement to tests only.

Reply:
you can add test deps that don't affect the package run deps

Reply:
Yes, thats what is done now.